moveit_core/collision_detection_fcl/include/moveit/collision_detection_fcl/collision_common.h

```diff
@@ -188,11 +191,10 @@ struct FCLGeometry
   }
 
   std::shared_ptr<fcl::CollisionGeometry> collision_geometry_;
-  boost::shared_ptr<CollisionGeometryData>  collision_geometry_data_;
+  CollisionGeometryDataPtr collision_geometry_data_;
 };
 
-typedef std::shared_ptr<FCLGeometry> FCLGeometryPtr;
-typedef std::shared_ptr<const FCLGeometry> FCLGeometryConstPtr;
+MOVEIT_CLASS_FORWARD(FCLGeometry);
```

Comment:
> oh, then why did you convert it recently to std?
> delftrobotics@7352495#diff-90ab8a9db10c5c40f0754a0638741a07L193

Last time round I saw 'fcl' and switched it. This time round I noticed it's purely a moveit class, so there's no requirement for it to be `std::shared_ptr`. I think it makes sense to use MOVEIT_CLASS_FORWARD for all moveit classes that need shared_ptr types, so I switched this one too. But like I said, I can undo that if preferred.

Regarding a MOVEIT_CLASS_FORWARD_STD, currently it would mainly be useful for FCL classes, which would mean putting the macro invocation inside the FCL namespace. Not sure if that would be a problem though.